From 732c6be972221a6efccf3f390ea78a093019a982 Mon Sep 17 00:00:00 2001
From: Robert Chiras <robert.chiras@nxp.com>
Date: Wed, 28 Nov 2018 15:09:56 +0200
Subject: [PATCH 5230/5242] Revert "MA-12957: arm64: dts: imx8qm/qxp mek:
 Correct interrupts for adv7535"

commit  cceeb839f4873a8c89e1bd6de71d035ad9f5455c from
https://source.codeaurora.org/external/imx/linux-imx.git

This commit is breaking suspend/resume, so reverting it until a fix is
provided.

This reverts commit 4a004884a4f16d698941a39fd0ec5f29bedb10ed.
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8qm-mek.dtsi  |    4 ++--
 arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek.dtsi |    4 ++--
 2 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek.dtsi
index c905071..ccb9460 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek.dtsi
@@ -593,7 +593,7 @@
 			fsl,pins = <
 				SC_P_MIPI_DSI0_I2C0_SCL_MIPI_DSI0_I2C0_SCL	0xc600004c
 				SC_P_MIPI_DSI0_I2C0_SDA_MIPI_DSI0_I2C0_SDA	0xc600004c
-				SC_P_MIPI_DSI0_GPIO0_01_MIPI_DSI0_GPIO0_IO01	0x00000020
+				SC_P_MIPI_DSI0_GPIO0_00_MIPI_DSI0_GPIO0_IO00	0x00000020
 			>;
 		};
 
@@ -601,7 +601,7 @@
 			fsl,pins = <
 				SC_P_MIPI_DSI1_I2C0_SCL_MIPI_DSI1_I2C0_SCL	0xc600004c
 				SC_P_MIPI_DSI1_I2C0_SDA_MIPI_DSI1_I2C0_SDA	0xc600004c
-				SC_P_MIPI_DSI1_GPIO0_01_MIPI_DSI1_GPIO0_IO01	0x00000020
+				SC_P_MIPI_DSI1_GPIO0_00_MIPI_DSI1_GPIO0_IO00	0x00000020
 			>;
 		};
 
diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek.dtsi
index a3ec242..772bd71 100755
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek.dtsi
@@ -456,7 +456,7 @@
 			fsl,pins = <
 				SC_P_MIPI_DSI0_I2C0_SCL_MIPI_DSI0_I2C0_SCL	0xc6000020
 				SC_P_MIPI_DSI0_I2C0_SDA_MIPI_DSI0_I2C0_SDA	0xc6000020
-				SC_P_MIPI_DSI0_GPIO0_01_MIPI_DSI0_GPIO0_IO01	0x00000020
+				SC_P_MIPI_DSI0_GPIO0_00_MIPI_DSI0_GPIO0_IO00	0x00000020
 			>;
 		};
 
@@ -470,7 +470,7 @@
 			fsl,pins = <
 				SC_P_MIPI_DSI1_I2C0_SCL_MIPI_DSI1_I2C0_SCL	0xc6000020
 				SC_P_MIPI_DSI1_I2C0_SDA_MIPI_DSI1_I2C0_SDA	0xc6000020
-				SC_P_MIPI_DSI1_GPIO0_01_MIPI_DSI1_GPIO0_IO01	0x00000020
+				SC_P_MIPI_DSI1_GPIO0_00_MIPI_DSI1_GPIO0_IO00	0x00000020
 			>;
 		};
 
-- 
1.7.9.5

